core/__tests__/actions/settings.ts

```diff
@@ -12,10 +14,33 @@ describe("actions/settings", () => {
       password: "P@ssw0rd!",
       email: "mario@example.com",
     });
+
+    connection = await specHelper.buildConnection();
+
+    await Setting.truncate();
+  });
+
+  test("can publicly read the cluster name", async () => {
+    const setting = await plugin.registerSetting(
+      "testPlugin",
+      "cluster-name",
+      "My Grouparoo Cluster",
+      "My Grouparoo Cluster",
+      "Name of the cluster",
+      "string"
+    );
```

Comment:
The setting should be created by default.  You may not need this.

Reply:
Put this in the code above it, just to be sure:

```js
const count = await Setting.count();
console.log("...", count);
```

And got back `0`. 

Also tried to add the `resetSettings: true` option to the test server in the helper, but that didn't change the result.

Is there something else I have to do to not need this code?

Reply:
I ended up deleting this code because it was testing the action that we don't need.